### ISSUE TITLE
Deprecation warning in `cellrank/tl/_colors.py`

### ISSUE THREAD

**Issue by WeilerP** [theislab/cellrank#682]
## Description

In `cellrank/tl/_colors.py:255`, the following `VisibleDeprecationWarning` is thrown:

```pytb
VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
```

See, for example, the testing stack trace in [this](https://github.com/theislab/cellrank/runs/3118375279?check_suite_focus=true) CI run.

**Comment by michalk8** [comment]
@Marius1311 could you take care of this? afaik you're most familiar with the `_map_names_and_colors` function.

**Comment by Marius1311** [comment]
Sure, I can do this.

**Comment by michalk8** [comment]
closed via #704
